Glossary of Terms.md

```diff
@@ -83,18 +83,30 @@ A [**payload**](#def-payload) containing one specific [HTTP Response](http://www
 ### Resource
 An API [**resource**](http://www.w3.org/TR/di-gloss/#def-resource) specified by its *URI*. It can also refer to a [**set of resources**](#def-resource) matching one [**URI template**](#def-uri-template).
 
+<a name="def-resource-group"></a>
+### Resource Group
+An arbitrary grouping of one or more [**Resource Sets**](#def-resource-set).
+
 <a name="def-resource-model"></a>
 ### Resource Model
 One [**manifestation of a resource**](http://www.w3.org/TR/di-gloss/#def-resource-manifestation) in the form of a [payload](#def-payload). A resource model is an example representation of its resource. Can be referenced later in the place of a [payload](#def-payload).
 
 <a name="def-resource-set"></a>
 ### Resource Set
-A set of API [**resources**](http://www.w3.org/TR/di-gloss/#def-resource) its *URI* matches one specific  [**URI template**](#def-uri-template).
+A set of API [**resources**](http://www.w3.org/TR/di-gloss/#def-resource) with *URI*, which matches one specific [**URI template**](#def-uri-template).
 
 <a name="def-trait"></a>
 ### Trait
 A quality or characteristic of an API Blueprint SECTION.
 
+<a name="def-transaction"></a>
+### Transaction
+[HTTP Request](#def-request) with a corresponding [HTTP Response](#def-response).
+
+<a name="def-transaction-example"></a>
+### Transaction Example
+An implicit group of one or more [Requests](#def-request) and one or more corresponding [Responses](#def-response). If expanded into Request-Response pairs, it represents one or more [**HTTP Transactions**](#def-transaction).
```

Comment:
I see the distinction. While Transaction Examples do not have any direct representation in the API Description Refract namespace (they're resolved into a list of Request-Response pairs), they still exist in API Blueprint document and it is important to understand them when writing the document.

In the specification, they are [mentioned just briefly](https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#example-multiple-transaction-examples). I wanted to recognize them here as an important term, to be able to link explanation of the term from other places.

However, since we are completely abandoning Transaction Examples in [blueprint-transactions](https://github.com/apiaryio/blueprint-transactions) at the end, I'm OK with not having this here at the moment.